react/week2/todoapp/src/components/TodoItem.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+
+const TodoItem = (props) => {
```

Comment:
That's more code style but since last class we looked at destructuring and that would fit very well here so you don't have to repeat "props.something" all across your component:
```suggestion
const TodoItem = ({ item, handleChange, removeTodo}) => {
```